internal/internal_decision_state_machine.go

```diff
@@ -375,6 +375,9 @@ func (d *decisionStateMachineBase) cancel() {
 		d.moveState(decisionStateCanceledBeforeInitiated, eventCancel)
```

Comment:
This turned out to be a non-dead code. At lease unit tests has some dependency on it.